templates/order/_ordered_items_table.html

```diff
@@ -51,9 +51,10 @@ <h3>
     <div class="table__row order-details__product{% if forloop.last %} order-details__last-row{% endif %}">
       <div class="row">
         <div class="col-md-6 col-9">
+          <p>{{item_line.product}}</p>
```

Comment:
We already display the product name (line 59), so I think this is not necessary.